### PR TITLE
[AppSec RN] Add REST API RN

### PR DESCRIPTION
**Pull request by anaghapamidi**

### CODE REVIEW

**Comment by Pubs-MV** [comment]
@anaghapamidi there are formatting issues with the row on API deprecation, please fix. Also the title for that row looks wrong. 
cc @manukumar6

File: cspm/rn/prisma-cloud-code-security-release-information/look-ahead-planned-updates-prisma-cloud-code-security.adoc [hunk 1]
```diff
@@ -45,9 +45,11 @@ NOTE: The details and functionality listed below are a preview and the actual re
 //RLP-112353
 | By the end of 2023 calendar year, BridgecrewCLI including GitHub Action, CircleCI Orb, and container, will be deprecated.  Plan to transition smoothly to Checkov and its compatible plugins to minimize disruption.
 
-|The following endpoints are deprecated in release 23.8.1 and will be sunset in release 23.9.1:
+|The following endpoints are deprecated in release 23.8.1 and will be sunset in release 23.9.2:
```
Comment (Pubs-MV) [inline cspm/rn/prisma-cloud-code-security-release-information/look-ahead-planned-updates-prisma-cloud-code-security.adoc:48]: Is this right heading?

File: cspm/rn/prisma-cloud-code-security-release-information/look-ahead-planned-updates-prisma-cloud-code-security.adoc [hunk 2]
```diff
@@ -45,9 +45,11 @@ NOTE: The details and functionality listed below are a preview and the actual re
...
 //BCE-17644
 
+|The following endpoints are deprecated in release 23.8.1 and will be sunset in release 23.9.2:
```
Comment (Pubs-MV) [inline cspm/rn/prisma-cloud-code-security-release-information/look-ahead-planned-updates-prisma-cloud-code-security.adoc]: Please review the formatting of the table. The replacement endpoints is hanging and is not displayed in the preview.

Fix needed to publish

File: cspm/rn/prisma-cloud-code-security-release-information/look-ahead-planned-updates-prisma-cloud-code-security.adoc (hunk 1 above)
Comment (anaghapamidi) [reply cspm/rn/prisma-cloud-code-security-release-information/look-ahead-planned-updates-prisma-cloud-code-security.adoc:48]: @manukumar6  - please help confirm

Comment (manukumar6) [reply cspm/rn/prisma-cloud-code-security-release-information/look-ahead-planned-updates-prisma-cloud-code-security.adoc:48]: Yes, Matangi. The complete note is there: https://github.com/PaloAltoNetworks/prisma-cloud-docs/blame/8c981c3c13804575fc62565e3a8f9d2ae7554ca9/cspm/rn/prisma-cloud-code-security-release-information/look-ahead-planned-updates-prisma-cloud-code-security.adoc

**Comment by anaghapamidi** [comment]
No more conflicts.

**Pull request merged**